src/map.rs

```diff
@@ -250,6 +279,15 @@ where
     /// Warning: `hash_builder` is normally randomly generated, and is designed to allow the map
     /// to be resistant to attacks that cause many collisions and very poor performance.
     /// Setting it manually using this function can expose a DoS attack vector.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use flurry::{HashMap, DefaultHashBuilder};
+    ///
+    /// let map = HashMap::with_capacity_and_hasher(10, DefaultHashBuilder::default());
```

Comment:
We could actually use the `RandomState` hasher in our example here. Using the default hasher seems a little odd, since it is the default after all :p

Reply:
You're right, this will better illustrate the function